tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -48,22 +56,35 @@ def _test_ios_packages(args):
         target_proj_path = os.path.join(temp_dir, 'ios_package_test')
         shutil.copytree(test_proj_path, target_proj_path)
 
-        # update the podspec to point to the local framework zip file
-        local_podspec_path = os.path.join(target_proj_path, 'onnxruntime-mobile.podspec')
-        local_podspec_template = os.path.join(target_proj_path, 'onnxruntime-mobile.podspec.template')
-        with open(local_podspec_template, 'r') as file:
-            file_data = file.read()
+        # generate the podspec file from the template
+        framework_info = load_framework_info(args.framework_info_file.resolve())
 
-        # replace the target strings
-        file_data = file_data.replace('${ORT_BASE_FRAMEWORK_ARCHIVE}', 'file:' + zip_file_path)
         with open(os.path.join(REPO_DIR, 'VERSION_NUMBER')) as version_file:
-            file_data = file_data.replace('${ORT_VERSION}', version_file.readline().strip())
+            ORT_VERSION = version_file.readline().strip()
+
+        variable_substitutions = {
+            "VERSION": ORT_VERSION,
+            "IOS_DEPLOYMENT_TARGET": framework_info["IOS_DEPLOYMENT_TARGET"],
+            "WEAK_FRAMEWORK": framework_info["WEAK_FRAMEWORK"],
+            "LICENSE_FILE": '"LICENSE"',
```

Comment:
does this file exist in the pod directory?
perhaps it doesn't matter for this test

Reply:
This file does not exist, and there is a Cocoapods warning for this when running the test, but seems harmless